src/main/java/edu/umich/its/cpm/BoxUtils.java

```diff
@@ -540,4 +525,78 @@ public static String getBoxClientRedirectUrl(HttpServletRequest request,
 		return env.getProperty(Utils.BOX_CLIENT_REDIRECT_URL) + "/authorized";
 	}
 
+	/**
+	 * add user as collaborate to Box folder
+	 * 
+	 * @param userEmail
+	 * @param role
+	 * @param folderId
+	 * @param boxClientId
+	 * @param boxClientSecret
+	 */
+	public static void addCollaboration(String boxAdminId, String userEmail,
+			String role, String folderId, String boxClientId,
+			String boxClientSecret) {
+		BoxAPIConnection api = getBoxAPIConnection(boxAdminId, boxClientId,
+				boxClientSecret);
+		if (api != null) {
+			try {
+				BoxFolder folder = new BoxFolder(api, folderId);
+				if ("Owner".equals(role)) {
+					// CTools Owner user role
+					folder.collaborate(userEmail, BoxCollaboration.Role.CO_OWNER);
```

Comment:
Could use if-else statement to just set the role and have a single call to folder.collaborate().